src/modules/commands/discord/urbandictionary.ts

```diff
@@ -0,0 +1,83 @@
+import { Command } from '@commands/CommandHandler'
+import got from 'got'
+import { isRecord } from '@app/validators/object'
+import camelcaseKeys from 'camelcase-keys'
+import discord from 'discord.js'
+
+type definition = {
+  definition: string
+  permalink: string
+  thumbsUp: number
+  soundUrls: string[]
+  author: string
+  word: string
+  defid: number
+  currentVote: string
+  writtenOn: string
+  example: string
+  thumbsDown: number
+}
+
+export default new Command<discord.Message>({
+  keyword: 'urbandictionary',
+  description: 'get the modern definition of a word or expression',
+  help: 'Usage: `{prefix}urbandictionary` - get the modern definition of a word or expression',
+  callback: async ({ args, reply, env }) => {
+    if (args.length === 0) return reply(env.command.getHelp(env.handler))
+
+    const word = args.join(' ') //Merge all the arguments for expressions
+
+    const term = word.toLowerCase()
```

Comment:
UrbanDictionary API is case sensitive, you shouldn't convert the term to lowercase.